### PR TITLE
Consolidate redundant backend creation in tests

### PR DESCRIPTION
**Pull request by Lewuathe**
# Purpose

Creating backend object every test cases is redundant. We can make test faster by eliminating this redundant object creation.

# Result

Before

```
Chrome 65.0.3325 (Mac OS X 10.12.6): Executed 4089 of 4089 SUCCESS (2 mins 18.853 secs / 2 mins 17.784 secs)
Firefox 58.0.0 (Mac OS X 10.12.0): Executed 4089 of 4089 SUCCESS (9 mins 7.102 secs / 8 mins 25.818 secs)
```

After

```
Chrome 65.0.3325 (Mac OS X 10.12.6): Executed 4089 of 4089 SUCCESS (1 min 15.038 secs / 1 min 1.975 secs)
Firefox 59.0.0 (Mac OS X 10.12.0): Executed 4089 of 4089 SUCCESS (5 mins 6.947 secs / 4 mins 25.21 secs)
```

---
This change is [Reviewable](https://reviewable.io/reviews/pair-code/deeplearnjs/861)

### CODE REVIEW

File: src/test_util.ts [hunk 1]
```diff
@@ -159,9 +162,12 @@ function executeTests(
 
     afterEach(() => {
       ENV.engine.endScope(null);
-      ENV.reset();
     });
 
+    afterAll(() => {
+      ENV.reset();
+    })
```
Comment (dsmilkov) [inline src/test_util.ts]: add `;`
so yarn lint passes: https://travis-ci.org/PAIR-code/deeplearnjs/builds/354896494#L643

**Comment by Lewuathe** [comment]
@dsmilkov Could you take a look when you get a chance?

**Comment by Lewuathe** [comment]
@dsmilkov @nsthorat Sorry bothering you. But could you take a look?
Since it's not the ops update, we can merge this even in API freeze, I think.

**Comment by dsmilkov** [comment]
:lgtm_strong: Thank you for this *amazing* contribution!

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/tensorflow/tfjs-core/861#-:-L9Lq_xZ-ZqAC4rSPPTA:b-gk86mx)*

**Comment by dsmilkov** [comment]
And thanks for the ping! We should appologize instead of you - it's our job to review the great contributions that our community makes to the library!

**Comment by Lewuathe** [comment]
Thank you so much!

**Pull request merged**